upgrade-2.x.sh

```diff
@@ -1291,6 +974,23 @@ FETCHED_SLUG=$(CURL_CA_BUNDLE="${TMPCRT}" ${CURL} -H "Authorization: Bearer ${AP
 SLUG=${FORCED_SLUG:-$FETCHED_SLUG}
 HOST_OS_VERSION=${META_BALENA_VERSION:-${VERSION_ID}}
 
+# Check host OS version
+case $VERSION in
+    [2-9].*|2[0-9][0-9][0-9].*.*)
+        if ! version_gt "$target_version" "$minimum_hostos_version" &&
+            ! [ "$target_version" == "$minimum_hostos_version" ]; then
+                log ERROR "Host OS version \"$VERSION\" too low, not supported."
+                else
+                log "Host OS version \"$VERSION\" OK."
```

Comment:
We have the exact same log right below right after the `fi`, so let's remove the duplicate one
```suggestion
```